docs/release/trg-0/trg-0-tractusx-metafile.md

```diff
@@ -0,0 +1,39 @@
+---
+title: TRG 0.0 - DRAFT tractusx metafile
```

Comment:
Perhaps "Repository metafile" is easier to understand?

Reply:
Agree, changes applied